### DISCUSSION TITLE
edit activity

### DISCUSSION THREAD

**Discussion by speising** [jlai/fitness-dashboard#56] [Ideas]
Hi, could you add the ability to edit activities? Especiallly important now that this no longer works from the app...

**Comment by jlai** [comment]
Unfortunately editing activities is not supported in the 3rd party web API.

**Comment by gramapp** [comment]
I was just about to ask about this. Activity editing is not working in the android app for me anymore. I guess just have to duplicate entries through this dashboard

**Comment by speising** [comment]
in theory, this could be simulated by deleting the old and creating an new activity, couldn't it?

**Reply by jlai** [reply]
Maybe for manually-created activities, but activities recorded by the device have a lot of other data attached to it that we can't create via the API.
